pages/dkp/konvoy/1.5/install/install-azure/index.md

```diff
@@ -56,7 +56,7 @@ This opens a browser window requesting your credentials. After supplying your cr
 If your account has access to many subscriptions, select a subscription to use. Enter the following command. It requires your subscription name. This is found in the output of the `az login` command.
 
   ```bash
-  az account set --subscription "YOUR_USER_LOGIN"
+  az account set --subscription "Your Subscription Label"
```

Comment:
At a glance `"Your Subscription Label"` looks like something I'm supposed to copy with the rest of the command, rather than a placeholder for a value I need to provide. I'm not sure if it's a mistake one's likely to make if they're paying close attention, but I could see myself missing it. Other commands in this doc use all-caps strings to indicate placeholder values, such as

```
az role assignment create --assignee YOUR_USER_LOGIN --role "User Access Administrator"
```